jccli/jc_api_v1.py

```diff
@@ -180,3 +177,71 @@ def set_user(self, username, attributes):
             body=Systemuserput(**attributes)
         )
         return api_response.to_dict()
+
+    def search_systems(self, filter={}):
+        """
+        Search for users on JumpCloud.
+
+        :param filter: (dict) an object used to filter search results for various fields. E.g.: `{"firstname": "David"}`
```

Comment:
can you please mention that search can be done on multiple fields and it's an AND filter? An example would be nice as well.

Reply:
Edited docstring to show that